Content.Shared/Examine/ExamineSystemShared.cs

```diff
@@ -54,6 +55,10 @@ public bool IsInDetailsRange(EntityUid examiner, EntityUid entity)
             if (IsClientSide(entity))
                 return true;
 
+            // Ghosts can see everything.
+            if (HasComp<GhostComponent>(examiner))
+                return true;
+
```

Comment:
If this is for performance you could slap in an EntityQuery<GhostComponent> too.

Reply:
True I did see this code showing up on CM.